integreat_cms/static/src/css/style.scss

```diff
@@ -228,7 +228,7 @@ select,
     @apply rounded text-blue-500 mr-2 align-text-top;
 
     &:disabled {
-        @apply bg-gray-100 border-gray-300;
+        @apply bg-gray-200 border-gray-200;
```

Comment:
```suggestion
        @apply bg-gray-200 border-gray-200;
        &:checked {
            @apply bg-gray-400 border-gray-400;
        }
```
I like the idea, but I noticed the contrast between a checked and unchecked disabled checkbox is extremely low, so I suggest to make checked disabled checkboxes a bit darker.

Reply:
I already committed another suggestion about the gray-scale from @JoeyStk.
Pls take a look at his comment above.
The last one wins or..? 😀

Reply:
Yes, I did see that, and I'm not contradicting this suggestion (using `gray-200` for both background and border looks fine to me), but I'd extend this to make sure the difference between checked disabled and unchecked disabled can be perceived...

| | Unchecked | Checked |
| --- | --- | --- |
| Before | ![Screenshot 2023-03-10 at 13-34-43 Integreat Redaktionssystem](https://user-images.githubusercontent.com/16021269/224317583-6c73d39b-d0e7-414c-bf37-666325d0b667.png)| ![Screenshot 2023-03-10 at 13-34-57 Integreat Redaktionssystem](https://user-images.githubusercontent.com/16021269/224317581-6e100757-8498-4fdb-9540-052553307244.png) |
| After @JoeyStk's suggestion | ![Screenshot 2023-03-10 at 13-33-16 Integreat Redaktionssystem](https://user-images.githubusercontent.com/16021269/224318020-9b83ba0c-e9a6-47d5-a87b-5002bc14a534.png) | ![Screenshot 2023-03-10 at 13-33-47 Integreat Redaktionssystem](https://user-images.githubusercontent.com/16021269/224318019-21a2c2de-e3b2-47d0-8b96-a066eb33cce6.png) |
| After my suggestion | ![Screenshot 2023-03-10 at 13-35-18 Integreat Redaktionssystem](https://user-images.githubusercontent.com/16021269/224317578-a785d411-8cd1-4628-a71b-40349c85a7a3.png) | ![Screenshot 2023-03-10 at 13-35-35 Integreat Redaktionssystem](https://user-images.githubusercontent.com/16021269/224317573-6ec01e57-6d02-4003-b0c8-f6da499b9064.png) |

Reply:
Ah sorry, I misread that